bookwyrm/models/book.py

```diff
@@ -299,6 +299,10 @@ class Edition(Book):
         max_length=255, choices=FormatChoices, null=True, blank=True
     )
     physical_format_detail = fields.CharField(max_length=255, blank=True, null=True)
+    audiobook_play_time = models.DurationField(
```

Comment:
Because this is calling a standard Django model field, rather than a custom BookWyrm activitypub model field, it's not going to be serialized in ActivityPub. To fix this, there needs to be a subclass of `DurationField` in https://github.com/bookwyrm-social/bookwyrm/blob/main/bookwyrm/models/fields.py#L528 -- that code will handle parsing the string version of the play time into a DurationField. 

I know this is kind of murky depths of how the activitypub implementation works --is that something you want to do, and if so does that give you enough of an idea of how to proceed? If you'd rather not, I can go ahead and make that change.

Reply:
@joachimesque Have you seen this?